data_structures/queue/queue.py

```diff
@@ -0,0 +1,69 @@
+from node import Node
+
+
+class Queue:
+
+    def __init__(self, iterable=[]):
+        """runs on initiation of class Queue"""
+        self.back = None
+        self._size = 0
+        
+        for item in iterable:
```

Comment:
You should have some sort of input validation to check that the passed in value for `iterable` is actually an iterable type.